sdks/go/pkg/beam/io/synthetic/source.go

```diff
@@ -165,10 +175,12 @@ type SourceConfigBuilder struct {
 func DefaultSourceConfig() *SourceConfigBuilder {
 	return &SourceConfigBuilder{
 		cfg: SourceConfig{
-			NumElements:   1, // 0 is invalid (drops elements).
-			InitialSplits: 1, // 0 is invalid (drops elements).
-			KeySize:       8, // 0 is invalid (drops elements).
-			ValueSize:     8, // 0 is invalid (drops elements).
+			NumElements:    1, // 0 is invalid (drops elements).
+			InitialSplits:  1, // 0 is invalid (drops elements).
+			KeySize:        8, // 0 is invalid (drops elements).
+			ValueSize:      8, // 0 is invalid (drops elements).
+			HotKeys:        2, // 0 is invalid (drops elements).
```

Comment:
Why "2"?

Reply:
By default, there should be no hot keys at all (every key should be unique).

Reply:
```
                         NumElements:    1, // 0 is invalid (drops elements).
			InitialSplits:  1, // 0 is invalid (drops elements).
			KeySize:        8, // 0 is invalid (drops elements).
			ValueSize:      8, // 0 is invalid (drops elements).
			NumHotKeys:     0,
			HotKeyFraction: 0,
```